### DISCUSSION TITLE
is the bam output from star-solo filtered?

### DISCUSSION THREAD

**Discussion by marlmatos** [alexdobin/STAR#2094] [Q&A] [answered]
Hi! 
I have been using STAR-solo with the WASP feature for allele-specific alignment. Since STAR does not do any filtering for the WASPout Tag "vW", I then cannot use the matrix.mtx that is outputed at the end of the "star solo align" workflow. 
I went ahead and filtered the *.aligned.sortedbycoordinate.bam file for "vW==1" and those reads not having the tag. Then, I tried to recreate the raw matrix.mtx with soloCOUNTmatrixFromBam.py and the filtered matrix with "STAR --runMode soloCellFiltering"

As I am doing this, I am wondering if the *.aligned.sortedbycoordinate.bam had any other sort of filtering like mapping quality, duplicates, etc?

and should i go back and refilter that?

**Comment by alexdobin** [comment, marked as answer]
Hi Marliette,

The BAM file is filtered in the standard way, i.e. with --outFilter* options.
